Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -200,6 +200,80 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
     }
 }
 
+/// An GRU cell.
+public struct GRUCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var updateWeight, updateWeight2: Tensor<Scalar>
+    public var resetWeight, resetWeight2: Tensor<Scalar>
+    public var outputWeight, outputWeight2: Tensor<Scalar>
+    public var updateBias, outputBias, resetBias: Tensor<Scalar>
+
+    @noDerivative public var stateShape: TensorShape {
+        [1, updateWeight.shape[0]]
+    }
+
+    public var zeroState: State {
+        State(hidden: Tensor(zeros: stateShape))
+    }
+
+    public typealias TimeStepInput = Tensor<Scalar>
+    public typealias TimeStepOutput = State
+    public typealias Input = RNNCellInput<TimeStepInput, State>
+    public typealias Output = RNNCellOutput<TimeStepOutput, State>
+
+    /// Creates a `GRUCell` with the specified input size and hidden state size.
+    ///
+    /// - Parameters:
+    ///   - inputSize: The number of features in 2-D input tensors.
+    ///   - hiddenSize: The number of features in 2-D hidden states.
+    public init(
+        inputSize: Int,
+        hiddenSize: Int,
+        seed: TensorFlowSeed = Context.local.randomSeed
+    ) {
+        let gateWeightShape = TensorShape([inputSize, 1])
+        let gateBiasShape = TensorShape([hiddenSize])
+        self.updateWeight = Tensor(glorotUniform: gateWeightShape, seed: seed)
+        self.updateWeight2 = Tensor(glorotUniform: gateWeightShape, seed: seed)
+        self.updateBias = Tensor(zeros: gateBiasShape)
+        self.resetWeight = Tensor(glorotUniform: gateWeightShape, seed: seed)
+        self.resetWeight2 = Tensor(glorotUniform: gateWeightShape, seed: seed)
+        self.resetBias = Tensor(zeros: gateBiasShape)
+        self.outputWeight = Tensor(glorotUniform: gateWeightShape, seed: seed)
+        self.outputWeight2 = Tensor(glorotUniform: gateWeightShape, seed: seed)
+        self.outputBias = Tensor(zeros: gateBiasShape)
+    }
+
+    public struct State: Differentiable {
+        public var hidden: Tensor<Scalar>
+
+        @differentiable
+        public init(hidden: Tensor<Scalar>) {
+            self.hidden = hidden
+        }
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameter input: The input to the layer.
+    /// - Returns: The hidden state.
+    @differentiable
+    public func callAsFunction(_ input: Input) -> Output {
+        let resetGate = sigmoid(matmul(input.input, resetWeight) +
+          matmul(input.state.hidden, resetWeight2) + resetBias)
```

Comment:
Indent each line wrapping by 4 from the previous line. The only difference from the Google Swift Style Guide is that we use 4-space indentation instead of 2-space.
```suggestion
            matmul(input.state.hidden, resetWeight2) + resetBias)
```
Same for other occurrences below.

Reply:
fixed 🙂